docker-compose.yml

```
@@ -0,0 +1,18 @@
+---
+version: "3.4"
+services:
+  server:
+    container_name: vaxx.link.server
+    build: ./server
+    ports:
+      # allow override of published port
+      - "127.0.0.1:${EXTERNAL_PORT:-9000}:8000"
+    env_file:
+      - .env
+  client:
+    container_name: vaxx.link.client
+    build: .
+    ports:
+      - "127.0.0.1:${EXTERNAL_PORT:-9001}:3000"
+    env_file:
+      - .env
```

Both NodeJS and [docker-compose](https://docs.docker.com/compose/environment-variables/#the-env-file) insist on using a config file named `.env`. 

I'd suggest using another name, just to make it clear where a config value is coming from.

```suggestion
      - app.env
```

---

Changed file name to the suggested `app.env` and added `default.app.env`